<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Инспектор аэропортов — MVP</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="gbSearch">
      <property name="title">
       <string>Параметры поиска</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lbFromTitle">
         <property name="text">
          <string>Откуда</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cbFrom"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lbToTitle">
         <property name="text">
          <string>Куда</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="cbTo"/>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="lbDateTitle">
         <property name="text">
          <string>Дата</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QDateEdit" name="deDate">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QPushButton" name="pbSearch">
         <property name="text">
          <string>Найти</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tvFlights"/>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="pbBuy">
        <property name="text">
         <string>Купить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbBookings">
        <property name="text">
         <string>Мои бронирования</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="sp1">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
